tools/pagerduty-migrator/migrator/resources/integrations.py

```diff
@@ -15,19 +15,20 @@ def match_integration(integration: dict, oncall_integrations: list[dict]) -> Non
 
 
 def match_integration_type(integration: dict, vendors: list[dict]) -> None:
-    vendors_map = {vendor["id"]: vendor for vendor in vendors}
-
-    if integration["type"] not in [
-        "generic_events_api_inbound_integration",
-        "events_api_v2_inbound_integration",
-    ]:
-        integration["oncall_type"] = None
-        return
-
-    vendor_id = integration["vendor"]["id"]
-    vendor_name = vendors_map[vendor_id]["name"]
-
-    integration["oncall_type"] = PAGERDUTY_TO_ONCALL_VENDOR_MAP.get(vendor_name)
+    if integration["vendor"] != None:
+        vendors_map = {vendor["id"]: vendor for vendor in vendors}
+    
+        if integration["type"] not in [
+            "generic_events_api_inbound_integration",
+            "events_api_v2_inbound_integration",
+        ]:
+            integration["name"] = None
```

Comment:
`integration["name"]` is really meant to hold the integration name, not the type/vendor, so I propose to change this back to `integration["oncall_type"]`.